cl/users/models.py

```diff
@@ -115,6 +119,9 @@ class UserProfile(models.Model):
         help_text="The user tests new features before they are finished",
         default=False,
     )
+    recap_email = models.EmailField(
```

Comment:
@mlissner Wanted to get some feedback on best approaches for this before I create a number of bad migrations. Problem:

1. recap_email is required and also unique.
2. We need some kind of placeholder for now, given the existing users.

### Options
1. Take away the unique constraint and have another placeholder that is meant to be overwritten by a job? Unique gets added later.
2. Allow blanks for now, take that away in a future migration. 
3. Do you have another pattern that deals with this more elegantly? As you can see I am using a post_save signal to handle it now.